web/src/components/molecules/Common/Form/AssetItem/index.tsx

```diff
@@ -69,14 +72,31 @@ const AssetItem: React.FC<Props> = ({
     fileList,
   };
 
+  useEffect(() => {
+    setAssetValue(assetList.find(asset => asset.id === defaultValue));
+  }, [defaultValue, assetList, setAssetValue]);
+
+  const getFile = (e: any) => {
+    console.log(e);
+  };
+
   return (
-    <Item name={name} label={label} extra={extra} rules={rules}>
-      <AssetButton onClick={handleClick}>
-        <div>
-          <Icon icon="link" />
-          <div style={{ marginTop: 8 }}>{t("Asset")}</div>
-        </div>
-      </AssetButton>
+    <Item name={name} label={label} extra={extra} rules={rules} getValueFromEvent={getFile}>
+      {assetValue ? (
+        <AssetButton onClick={handleClick}>
+          <div>
+            <Icon icon="folder" size={24} />
+            <div style={{ marginTop: 8, overflow: "hidden" }}>{assetValue.fileName}</div>
+          </div>
+        </AssetButton>
+      ) : (
+        <AssetButton onClick={handleClick}>
+          <div>
+            <Icon icon="link" />
+            <div style={{ marginTop: 8 }}>{t("Asset")}</div>
+          </div>
+        </AssetButton>
+      )}
```

Comment:
The styles here are different than the design. The design has different icons and the border is dashed

Reply:
@KaWaite , changed.